sygnal/gcmpushkin.py

```diff
@@ -251,8 +251,8 @@ async def _request_dispatch(self, n, log, body, headers, pushkeys, span):
             return pushkeys, []
         elif 200 <= response.code < 300:
             try:
-                resp_object = json.loads(response_text)
-            except JSONDecodeError:
```

Comment:
This is causing flake8 to fail, since the import is still present on line 21. (`F401 'json.JSONDecodeError' imported but unused`)

Reply:
Fixed and pushed to your branch